pubspec.yaml

```diff
@@ -58,6 +58,7 @@ dependencies:
   flutter_math_fork: ^0.5.0
   scroll_to_index: ^2.0.0
   showcaseview: ^1.1.4
+  sizer: ^2.0.15
```

Comment:
`sizer` is not used in your changes, do you intent to use it ?

Reply:
It is not used cause of errors while testing. But, we can surely use it for future purposes or at other required sizings to make the app more issue-proof. Your views on it?

Reply:
I don't think any unused package should be defined without any purpose. If it's required in future, it can be added at that time itself.

Reply:
Sure removing it! 👍